tests/Hosting.Services.Web.UnitTests/Middlewares/ResponseHeadersMiddlewareTests.cs

```diff
@@ -15,6 +16,10 @@ namespace Hosting.Services.Web.UnitTests
 	[TestClass]
 	public class ResponseHeadersMiddlewareTests
 	{
+		private const string MachineHeaderName = "X-Machine";
+		private const string BuildVersionHeaderName = "X-Machine";
+		private const string TraceIdHeaderName = "X-Machine";
```

Comment:
It looks like thos two consts need to have diffirent values, or maybe get it from the cluss you are testing, by making them internal there.

Reply:
Yeah thanks, I did not push latest changes :/

Reply:
Fixed now